projects/polynomial/index.js

```diff
@@ -7,6 +7,12 @@ const chain = 'optimism'
 const { getConfig } = require('../helper/cache')
 const transform = t => `${chain}:${t}`
 
+// api
+const BASE_URL = "https://perps-api-experimental.polynomial.fi/snx-perps/tvl";
+const api = axios.create({
```

Comment:
can this somehow be be fetched on chain?

Reply:
Each user has a Smart Wallet.

First you'd need to get all of them:
https://optimistic.etherscan.io/address/0xd567e18fdf8afa58953dd8b0c1b6c97adf67566b#readContract
accountAddr(id) up to id = accounts (currently 6k+)

Then for each wallet check sUSD balance and how much margin is available on every perp market (currently 24) for this wallet